Dockerfile

```diff
@@ -26,12 +26,13 @@ WORKDIR /usr/src/ipoib-cni
 RUN make clean && \
     make build
 
-FROM alpine:3
+FROM nvcr.io/nvidia/doca/doca:3.0.0-base-rt-host
 COPY --from=builder /usr/src/ipoib-cni/build/ipoib /usr/bin/
 WORKDIR /
 
-LABEL io.k8s.display-name="IPoIB CNI"
+LABEL https://nvcr.io/nvidia/cloud-native/IPoIB-CNI
```

Comment:
while the old label propably didnt mean much, im not sure what is the meaning of the new label

here is a set of well known labels:
https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys

apart of that it can be any key=value pair AFAIK